src/systems/user_commands/UserCommands.cc

```diff
@@ -859,7 +871,6 @@ void UserCommandsPrivate::OnCmdLight(const msgs::Light &_msg)
   }
 }
 
-
```

Comment:
nit: unnecessary whitespace change

Reply:
My eyes didn't like the lack of uniformity by leaving the extra space. Haha
![image](https://github.com/gazebosim/gz-sim/assets/10233412/163ff593-7d9d-4954-9305-2066f614fd74)

Reply:
Feel free to re-open if change needed.